<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2241</width>
    <height>1946</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cbe5f2
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>351</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Vaccine Name&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Manufacturing Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Expiration Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Supplier&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Batch Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Stock Quantity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>500</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>220</y>
       <width>110</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2025</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>290</y>
       <width>110</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2025</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>590</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: #144976</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>630</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>360</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>630</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>150</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>430</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Pfizer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GlaxoSmithKline</string>
      </property>
     </item>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_4</zorder>
    <zorder>label_9</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>spinBox</zorder>
    <zorder>dateEdit</zorder>
    <zorder>dateEdit_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>comboBox</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>radioButton</zorder>
    <zorder>radioButton_2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>1211</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>1151</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: #144976</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID Vaccine</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vaccine Name</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Manufacturing Date</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expiration Date</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Batch Number</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Supplier</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stock Quantity</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #144976
</string>
    </property>
    <property name="text">
     <string>research</string>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/chercher.png</normaloff>:/new/prefix1/chercher.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #144976
</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>search by  </string>
    </property>
    <item>
     <property name="text">
      <string>search by</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID Vaccine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vaccine Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manufacturing Date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Expiration Date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Batch Number</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Supplier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stock Quantity</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>77</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #144976
</string>
    </property>
    <property name="placeholderText">
     <string>sort by </string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>sort by</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vaccine Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manufacturing Date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Expiration Date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stock Quantity</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #144976</string>
    </property>
    <property name="text">
     <string>Log-out</string>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/se-deconnecter (1).png</normaloff>:/new/prefix1/se-deconnecter (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #144976
</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/maison.png</normaloff>:/new/prefix1/maison.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #144976
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic;&quot;&gt;Vaccine Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radious:5ppx;</string>
    </property>
    <property name="placeholderText">
     <string>Enter search text...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/telecharger-le-pdf.png</normaloff>:/new/prefix1/telecharger-le-pdf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>958</x>
      <y>115</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/tourner.png</normaloff>:/new/prefix1/tourner.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>83</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/graphique.png</normaloff>:/new/prefix1/graphique.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/sombre.png</normaloff>:/new/prefix1/sombre.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>520</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
    background-color: #dc3545</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/alarme.png</normaloff>:/new/prefix1/alarme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>83</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/a.qrc">
      <normaloff>:/new/prefix1/gestion-du-temps.png</normaloff>:/new/prefix1/gestion-du-temps.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>20</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images/a.qrc">:/new/prefix1/p1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>comboBox_4</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_10</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images/a.qrc"/>
 </resources>
 <connections/>
</ui>
